### DISCUSSION TITLE
After update to 0.5.1, cannot login

### DISCUSSION THREAD

**Discussion by dietrichmd** [mealie-recipes/mealie#567] [Q&A]
Just updated to 0.5.1 and now I can't login.  Any suggestions?  UN/PW was autosaved in browser and are both correct.

**Comment by sephrat** [comment]
If you upgraded from 0.4.x or below, your database has not been migrated as the [release note explained](https://hay-kot.github.io/mealie/changelog/v0.5.0/). You need to use the [default password](https://hay-kot.github.io/mealie/documentation/admin/user-management/#startup) and restore from your previous backup.

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm closing it since it's no longer relevant. If you're having trouble migrating to the newest version, feel free to reach out here or on Discord.
